polyfills/Array/from/polyfill.js

```diff
@@ -1,36 +1,104 @@
-Object.defineProperty(Array, 'from', {
-	configurable: true,
-	value: function from(source) {
-		// handle non-objects
-		if (source === undefined || source === null) {
-			throw new TypeError(source + ' is not an object');
+(function arrayFromPolyfill() {
```

Comment:
Good catch @triblondon it leaks in IE6-8. removing the name, leaving wrapped in anonymous. IIFE